### PR TITLE
WRN-12923: Added `icon` public class name to `Icon` component

### PR DESCRIPTION
**Pull request by sjro**
Enact-DCO-1.0-Signed-off-by: SJ RO (sj.ro@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Sandstone Icon needs to support custom styles.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed `css` prop to public and added public class name `icon`.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-12923

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Added
 
+- `sandstone/Icon` public classname `icon`
```
Comment (MikyungKim) [inline CHANGELOG.md:11]:
```suggestion
- `sandstone/Icon` public class name `icon`
```

File: Icon/Icon.js [hunk 2]
```diff
@@ -98,7 +103,8 @@ const IconBase = kind({
 	},
 
 	styles: {
-		css: componentCss
+		css: componentCss,
+		publicClassNames:['icon']
```
Comment (MikyungKim) [inline Icon/Icon.js:107]:
```suggestion
		publicClassNames: ['icon']
```

**Review by MikyungKim — COMMENTED** [review]
Please update your branch to the recent develop.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**